criterion-measurement/src/Criterion/Types/Internal.hs

```diff
@@ -0,0 +1,60 @@
+{-# LANGUAGE BangPatterns #-}
```

Comment:
Now that this lives here, there's no need for the `Criterion.Types.Internal` in `criterion`, is there? Delete it.